.github/actions/javascript/createOrUpdateStagingDeploy/createOrUpdateStagingDeploy.ts

```diff
@@ -55,23 +55,40 @@ async function run(): Promise<IssuesCreateResponse | void> {
         const currentChecklistData: StagingDeployCashData | undefined = shouldCreateNewDeployChecklist ? undefined : GithubUtils.getStagingDeployCashData(mostRecentChecklist);
 
         // Find the list of PRs merged between the current checklist and the previous checklist
-        const mergedPRs = await GitUtils.getPullRequestsMergedBetween(previousChecklistData.tag, newStagingTag);
+        const mergedPRs = await GitUtils.getPullRequestsDeployedBetween(previousChecklistData.tag, newStagingTag);
+
+        // mergedPRs includes cherry-picked PRs that have already been released with previous checklist, so we need to filter these out
+        const previousPRNumbers = new Set(previousChecklistData.PRList.map((pr) => pr.number));
+        core.startGroup('Filtering PRs:');
+        core.debug('mergedPRs includes cherry-picked PRs that have already been released with previous checklist, so we need to filter these out');
```

Comment:
I wouldn't recommend using `core.debug`, because they'll only show if you re-run a workflow with debug logs enabled. They don't show at all by default

Reply:
Ah thanks!